bin/monit-dashboard.py

```diff
@@ -64,35 +66,45 @@ def getMonit():
                 checks[name] = status[name]
 
             sorted_checks = OrderedDict()
-            sorted_checks = OrderedDict(sorted(checks.iteritems(), key=itemgetter(1), reverse=True))
+            sorted_checks = OrderedDict(sorted(checks.iteritems(),
+                key=itemgetter(1), reverse=True))
             count = calculate_count(sorted_checks)
-            server = dict(name=site, url=s['url'], result=sorted_checks, s_rate=count)
+            server = dict(name=site, url=s['url'],
+                result=sorted_checks, s_rate=count)
 
             output.append(server)
-
     print(datetime.datetime.now())
     return(output)
 
 # Classes
 
 class monitDashboard(web.application):
-
     def run(self, port=8080, *middleware):
         func = self.wsgifunc(*middleware)
         return web.httpserver.runsimple(func, ('0.0.0.0', port))
 
 
 class index(object):
-
     def GET(self):
-        return render.index(output=getMonit(), now=datetime.datetime.now())
-
+        return render.index(output=getMonit(),
+            now=datetime.datetime.now(),
+            download_api=os.environ['DOWNLOAD_API_URL'])
```

Comment:
If it would be possible to replace it with a path (let's say, `/download` instead of a complete URL, maybe the environment variable wouldn't be needed (and eventually the `import os` in `bin/utils.py:2` too)

Reply:
@adriaaah  yes, its too simple but I did so much over-engineering coz I m new to frond-end stuffs.
`import os` required to verify the file exists.